autoscheduler/frontend/src/types.ts

```diff
@@ -1,3 +1,132 @@
+/* eslint-disable max-classes-per-file */
```

Comment:
Honestly, I think these classes should be separated into different files, especially since they have their own logic in them now(with `Meeting` and `MeetingType` being in one).

While I do think `max-classes-per-file`'s default of 1 is a bit too strict(which I propose to change to 2), I do think it's correct in indicating that this file is falling outside of the guideline for giving a file a single responsibility.

Reply:
1 class per file should be fine, since `MeetingType` is an enum and doesn't count as a class